### PR TITLE
Brig Physician

### PR DESCRIPTION
**Pull request by TheCaramelion**
## About The Pull Request

Med-Sec hybrid role, they're the ones taking care of hurt officers and whoever is in the brig. Not supposed to arrest people, or else they'll get a talk from the HoS.

## Changelog
:cl:
add: Brig Physician role
fix: Geneticists start with their right headset now
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/id cards/CH_ids.dm [hunk 1]
```diff
@@ -35,4 +35,8 @@
 	name = "casino manager id"
 	desc = "An ID card for the manager of the golden goose casino, fancy!"
 	icon_state = "ch_casino_manager"
-	access = list(160, 200, 201, 202, 203, 300)
\ No newline at end of file
+	access = list(160, 200, 201, 202, 203, 300)
+
+/obj/item/weapon/card/id/security/brigphysician
+	name = "Brig Physician's ID"
```
Comment (Kashargul) [inline code/game/objects/items/weapons/id cards/CH_ids.dm:41]: should use the define

File: modular_chomp/code/game/objects/items/devices/radio/encryptionkey.dm [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/obj/item/device/encryptionkey/headset_medsec
+	name = "medical security radio encryption key"
+	icon_state = "medsec_cypherkey"
+	channels = list("Medical" = 1, "Security" = 1)
```
Comment (Kashargul) [inline modular_chomp/code/game/objects/items/devices/radio/encryptionkey.dm:4]: should use the defines

File: modular_chomp/code/game/objects/items/devices/radio/headset.dm [hunk 3]
```diff
@@ -93,3 +93,9 @@
 	icon_state = "sec_cypherkey"
 
 	channels = list("Outsider" = 1)
```
Comment (Kashargul) [inline modular_chomp/code/game/objects/items/devices/radio/headset.dm:95]: Seems I missed that, can likely be fixed along as well

File: modular_chomp/code/game/jobs/job/security.dm [hunk 4]
```diff
@@ -0,0 +1,39 @@
+/datum/job/brigphysician
+	title = JOB_SECURITY_BRIGPHYS
+	flag = BRIGPHYS
+	departments = list(DEPARTMENT_SECURITY, DEPARTMENT_MEDICAL)
+	department_flag = ENGSEC
```
Comment (Kashargul) [inline modular_chomp/code/game/jobs/job/security.dm:5]: Engineering Security?

**Review by Kashargul — APPROVED** [review]

**Comment by Kazzc** [comment]
Fun, this role will make more sense later on. . . No spoilers~

**Pull request merged**